### ISSUE TITLE
[Bug] dash. does not show disk usage 

### ISSUE THREAD

**Issue by BadCactus634** [MauriceNino/dashdot#292]
### Description of the bug

In the storage section, all my disks are shown as 0% used.

### Info output of dashdot cli

```shell
yarn run v1.22.19
$ node dist/apps/cli/main.js info
INFO
=========
Yarn: 1.22.19
Node: v18.7.0
Dash: 4.3.10

Cwd: /app
Hash: 3caa68f5ee9bfdf14dadcd5c5a0358aadadff90b
In Docker: true
In Podman: false
In Docker (env): true
Done in 0.35s.
```

### What browsers are you seeing the problem on?

Chrome

### Where is your instance running?

Linux Server

### Additional context

I mounted the single volumes in the docker-compose file, they are recognized right, but the usage is not showing up.

This is the volumes part of my docker-compose file:

      - /etc/os-release:/mnt/host/etc/os-release:ro
      - /proc/1/ns/net:/mnt/host/proc/1/ns/net:ro
      - /:/mnt/host/sda:ro
      - /mnt/500GB_A:/mnt/host/sdb:ro
      - /mnt/1TB_A:/mnt/host/sdc:ro
      - /mnt/500GB_B:/mnt/host/sdd:ro

**Comment by MauriceNino** [comment]
Hey, thanks for creating an issue.
It seems like you tried to manually set up the config, but I think in your case the mounts should be this:

```
  - /:/mnt/host:ro
```

You are already mounting the host root into the container anyway, so no point in doing all the extra manual work.

---

After you are done with that change, can you please send me the output of the following command?

```
docker exec yarn cli raw-data --storage
```

**Comment by BadCactus634** [comment]
I already tried to, but then it shows usage only for some disks:
![image](https://user-images.githubusercontent.com/68558172/185760126-b8c51d5a-d043-460b-92cf-1e42af75f83c.png)

This is the output:
`yarn run v1.22.19
$ node dist/apps/cli/main.js raw-data --storage
Disk Layout: [
  {
    device: '/dev/sda',
    type: 'SSD',
    name: 'KINGSTON SA400S3',
    vendor: 'Kingston Technology',
    size: 240057409536,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '0100',
    serialNum: '',
    interfaceType: 'SATA',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/sdb',
    type: 'HD',
    name: 'WDC WD5000AAKX-6',
    vendor: 'Western Digital',
    size: 500107862016,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '1H18',
    serialNum: '',
    interfaceType: 'SATA',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/sdc',
    type: 'HD',
    name: 'WDC WD10EZEX-08W',
    vendor: 'Western Digital',
    size: 1000204886016,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '1A02',
    serialNum: '',
    interfaceType: 'SATA',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/sdd',
    type: 'HD',
    name: 'WDC WD5000AAKX-2',
    vendor: 'Western Digital',
    size: 500107862016,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '1H17',
    serialNum: '',
    interfaceType: 'SATA',
    smartStatus: 'unknown',
    temperature: null
  }
]
FS Size: [
  {
    fs: 'overlay',
    type: 'overlay',
    size: 235148750848,
    used: 110298296320,
    available: 112830996480,
    use: 49.43,
    mount: '/'
  },
  {
    fs: '/dev/sda2',
    type: 'ext4',
    size: 235148750848,
    used: 110298296320,
    available: 112830996480,
    use: 49.43,
    mount: '/mnt/host'
  },
  {
    fs: '/dev/sdd1',
    type: 'ext4',
    size: 491110858752,
    used: 32112738304,
    available: 433976004608,
    use: 6.89,
    mount: '/mnt/host/mnt/200GB'
  },
  {
    fs: '/dev/loop1',
    type: 'squashfs',
    size: 58327040,
    used: 58327040,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/core18/2409'
  },
  {
    fs: '/dev/loop0',
    type: 'squashfs',
    size: 58327040,
    used: 58327040,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/core18/2538'
  },
  {
    fs: '/dev/loop2',
    type: 'squashfs',
    size: 65011712,
    used: 65011712,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/core20/1611'
  },
  {
    fs: '/dev/loop4',
    type: 'squashfs',
    size: 71172096,
    used: 71172096,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/lxd/22753'
  },
  {
    fs: '/dev/loop5',
    type: 'squashfs',
    size: 73793536,
    used: 73793536,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/lxd/21029'
  },
  {
    fs: '/dev/loop3',
    type: 'squashfs',
    size: 65011712,
    used: 65011712,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/core20/1593'
  },
  {
    fs: '/dev/loop7',
    type: 'squashfs',
    size: 49283072,
    used: 49283072,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/snapd/16010'
  },
  {
    fs: '/dev/loop6',
    type: 'squashfs',
    size: 49283072,
    used: 49283072,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/snapd/16292'
  },
  {
    fs: '/dev/sdb1',
    type: 'ext4',
    size: 491110858752,
    used: 331505643520,
    available: 134583099392,
    use: 71.13,
    mount: '/mnt/host/mnt/500GB_A'
  },
  {
    fs: '/dev/sdc1',
    type: 'ext4',
    size: 983358451712,
    used: 443730186240,
    available: 489601335296,
    use: 47.54,
    mount: '/mnt/host/mnt/1TB_A'
  }
]
Block Devices: [
  {
    name: 'sda',
    type: 'disk',
    fsType: '',
    mount: '',
    size: 240057409536,
    physical: 'SSD',
    uuid: '',
    label: '',
    model: 'KINGSTON SA400S3',
    serial: '',
    removable: false,
    protocol: 'sata',
    group: undefined
  },
  {
    name: 'sdb',
    type: 'disk',
    fsType: '',
    mount: '',
    size: 500107862016,
    physical: 'HDD',
    uuid: '',
    label: '',
    model: 'WDC WD5000AAKX-6',
    serial: '',
    removable: false,
    protocol: 'sata',
    group: undefined
  },
  {
    name: 'sdc',
    type: 'disk',
    fsType: '',
    mount: '',
    size: 1000204886016,
    physical: 'HDD',
    uuid: '',
    label: '',
    model: 'WDC WD10EZEX-08W',
    serial: '',
    removable: false,
    protocol: 'sata',
    group: undefined
  },
  {
    name: 'sdd',
    type: 'disk',
    fsType: '',
    mount: '',
    size: 500107862016,
    physical: 'HDD',
    uuid: '',
    label: '',
    model: 'WDC WD5000AAKX-2',
    serial: '',
    removable: false,
    protocol: 'sata',
    group: undefined
  },
  {
    name: 'loop0',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/core18/2538',
    size: 58261504,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop1',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/core18/2409',
    size: 58232832,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop2',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/core20/1611',
    size: 64966656,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop3',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/core20/1593',
    size: 64970752,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop4',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/lxd/22753',
    size: 71106560,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop5',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/lxd/21029',
    size: 73728000,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop6',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/snapd/16292',
    size: 49242112,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop7',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/snapd/16010',
    size: 49233920,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'sda1',
    type: 'part',
    fsType: '',
    mount: '',
    size: 1048576,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'sda2',
    type: 'part',
    fsType: 'ext4',
    mount: '/mnt/host',
    size: 240053649408,
    physical: '',
    uuid: '086b566c-905f-4ce4-b169-b2928891c753',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'sdb1',
    type: 'part',
    fsType: 'ext4',
    mount: '/mnt/host/mnt/500GB_A',
    size: 500106788864,
    physical: '',
    uuid: '4c28e3f5-70eb-49ca-8d98-e7f8479dc1ae',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'sdc1',
    type: 'part',
    fsType: 'ext4',
    mount: '/mnt/host/mnt/1TB_A',
    size: 1000203091968,
    physical: '',
    uuid: '360b4050-8777-4bd5-84ff-5cf7dc8601d5',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'sdd1',
    type: 'part',
    fsType: 'ext4',
    mount: '/mnt/host/mnt/500GB_B',
    size: 500106788864,
    physical: '',
    uuid: 'b5ec1270-073b-474a-8359-55a19964293f',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  }
]
Done in 0.28s.`

If I do **df** the system correctly shows all my HDD with their usage.

**Comment by MauriceNino** [comment]
:tada: This issue has been resolved in version 4.4.0

Please check the [changelog](https://github.com/MauriceNino/dashdot/blob/main/.github/CHANGELOG.md) for more details.
